lib/matplotlib/text.py

```diff
@@ -2072,4 +2083,29 @@ def get_tightbbox(self, renderer=None):
         return super().get_tightbbox(renderer)
 
 
+    def _check_xytext(self, renderer=None):
```

Comment:
I would put this method just after the `_check_xy` method, since they fulfil related purposes.